src/ascend.js

```diff
@@ -0,0 +1,25 @@
+var _curry3 = require('./internal/_curry3');
+
+/**
+ * Makes an ascending comparator function out of a function that returns a value
+ * that can be compared with `<` and `>`.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig Ord b => (a -> b) -> a -> a -> Number
+ * @param {Function} fn A function of arity one that returns a value that can be compared
+ * @return {Function} A Function :: a -> b -> Int that returns `-1` if a < b, `1` if b < a, otherwise `0`
```

Comment:
Although I suppose we could make it ternary, that isn't the expected API.  The point is the signature as it stands, or better, as it should be cleaned up:

```hs
comparator:: ((a, a) -> Bool) -> ((a, a) -> Int)
-- or
comparator:: (a -> a -> Bool) -> (a -> a -> Int)
```

We accept a binary predicate (which really doesn't need to be curried, although it won't hurt if it is) and return a binary function that returns 0, -1, or +1, depending upon the values of `pred(first, second)` and `pred(second, first)`.  In other words, this function _creates_ a comparator suitable for use in `R.sort` or `Array.prototype.sort`.

Note that this is essentially using the predicated to extend the notion of `<` to additional types, although we don't have language level operator overloading.